### PR TITLE
MAN-77 Create Policy model, views and controller

### PR DESCRIPTION
**Pull request by nomadicoder**
Issue #MAN-77

- Administrate policy
- Create populate task
- Attach policy to building 
- Attach policy to space
- Attach policy to group

### CODE REVIEW

File: app/controllers/admin/policies_controller.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#frozen_string_literal: true
+
+module Admin
+  class PoliciesController < Admin::ApplicationController
+    def create
+      policy_params = params[:policy]
```
[the remark this answers is not in the document]
Comment (bibliotechy) [reply app/controllers/admin/policies_controller.rb]: @skng5 still relevant

File: app/models/group.rb [hunk 2]
```diff
@@ -23,6 +24,8 @@ class Group < ApplicationRecord
   has_many :service_group
   has_many :related_services, through: :service_group, source: :service
 
+  has_many :policies, as: :policy_makeable
```
Comment (bibliotechy) [inline app/models/group.rb]: I think this needs to  be removed

File: app/models/building.rb [hunk 3]
```diff
@@ -14,6 +14,8 @@ class Building < ApplicationRecord
 
   has_one_attached :photo, dependent: :destroy
 
+  has_many :policies, as: :policy_makeable
```
Comment (bibliotechy) [inline app/models/building.rb]: I think this needs to be removed. And shoudl building `include HasPolicies`

File: spec/factories/building_policies.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```
Comment (bibliotechy) [inline spec/factories/building_policies.rb]: Delete this file

File: spec/factories/space_policies.rb [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```
Comment (bibliotechy) [inline spec/factories/space_policies.rb]: Delete this file

File: spec/factories/group_policies.rb [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```
Comment (bibliotechy) [inline spec/factories/group_policies.rb]: Delete this file

**Comment by bibliotechy** [comment]
@skng5 OK, reviewed your changes. Please see the second review.

File: app/controllers/admin/policies_controller.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (nomadicoder) [reply app/controllers/admin/policies_controller.rb]: @bibliotechy - Deleted...

File: app/models/building.rb (hunk 3 above)
Comment (nomadicoder) [reply app/models/building.rb]: Done

File: app/models/group.rb (hunk 2 above)
Comment (nomadicoder) [reply app/models/group.rb]: done

**Comment by nomadicoder** [comment]
@bibliotechy Ready to review.

**Pull request merged**